src/compas/artists/meshartist.py

```diff
@@ -360,17 +318,17 @@ def draw_faces(self, faces=None, color=None, text=None):
         """
         raise NotImplementedError
 
-    @abstractmethod
-    def draw_mesh(self):
-        """Draw the mesh of the mesh.
+    # @abstractmethod
```

Comment:
aha! got it now... 👍 sounds perfect then..! it would still make sense to include a deprecated alias `draw_mesh` to not break the significant bunch of code and examples that use it.